### ISSUE TITLE
Latest versions failing on Mac

### ISSUE THREAD

**Issue by agzam** [snoe/clojure-lsp#82]
I've tried downloading two latest versions, and trying to connect to the server in Emacs fails for both of them, as well with `brew install --HEAD` version. Only the stable brew version `clojure-lsp: stable 20190408` works, the others fail with the following message:
```
Picked up JAVA_TOOL_OPTIONS: -Dapple.awt.UIElement=true
Nov 03, 2019 11:43:26 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: An error occurred while processing an incoming message.
java.lang.NullPointerException
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:275)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:186)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:188)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:90)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:95)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by agzam** [comment]
the contents of /tmp/lsp.out:

```
INFO  clojure-lsp.main: Server started
INFO  clojure-lsp.main: ====== LSP nrepl server started on port 50284
WARN  clojure-lsp.main: Initialize
ERROR clojure-lsp.main: #error {
 :cause nil
 :via
 [{:type java.lang.NullPointerException
   :message nil
   :at [sun.nio.fs.UnixPath toUnixPath UnixPath.java 198]}]
 :trace
 [[sun.nio.fs.UnixPath toUnixPath UnixPath.java 198]
  [sun.nio.fs.UnixPath resolve UnixPath.java 397]
  [sun.nio.fs.UnixPath resolve UnixPath.java 43]
  [sun.reflect.NativeMethodAccessorImpl invoke0 NativeMethodAccessorImpl.java -2]
  [sun.reflect.NativeMethodAccessorImpl invoke NativeMethodAccessorImpl.java 62]
  [sun.reflect.DelegatingMethodAccessorImpl invoke DelegatingMethodAccessorImpl.java 43]
  [java.lang.reflect.Method invoke Method.java 498]
  [clojure.lang.Reflector invokeMatchingMethod Reflector.java 93]
  [clojure.lang.Reflector invokeInstanceMethod Reflector.java 28]
  [clojure_lsp.crawler$to_file invokeStatic crawler.clj 23]
  [clojure_lsp.crawler$to_file invoke crawler.clj 22]
  [clojure_lsp.crawler$try_project invokeStatic crawler.clj 179]
  [clojure_lsp.crawler$try_project invoke crawler.clj 178]
  [clojure_lsp.crawler$get_project_from$fn__11762 invoke crawler.clj 194]
  [clojure.lang.PersistentVector reduce PersistentVector.java 341]
  [clojure.core$reduce invokeStatic core.clj 6747]
  [clojure.core$reduce invoke core.clj 6730]
  [clojure_lsp.crawler$get_project_from invokeStatic crawler.clj 192]
  [clojure_lsp.crawler$get_project_from invoke crawler.clj 191]
  [clojure_lsp.crawler$determine_dependencies invokeStatic crawler.clj 208]
  [clojure_lsp.crawler$determine_dependencies invoke crawler.clj 202]
  [clojure_lsp.handlers$initialize invokeStatic handlers.clj 97]
  [clojure_lsp.handlers$initialize invoke handlers.clj 87]
  [clojure.lang.Var invoke Var.java 390]
  [clojure_lsp.main$fn__13481$fn__13482 invoke main.clj 283]
  [clojure_lsp.main.proxy$clojure_lsp.ClojureExtensions$LanguageServer$c8d5825a initialize nil -1]
  [sun.reflect.NativeMethodAccessorImpl invoke0 NativeMethodAccessorImpl.java -2]
  [sun.reflect.NativeMethodAccessorImpl invoke NativeMethodAccessorImpl.java 62]
  [sun.reflect.DelegatingMethodAccessorImpl invoke DelegatingMethodAccessorImpl.java 43]
  [java.lang.reflect.Method invoke Method.java 498]
  [org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint lambda$null$0 GenericEndpoint.java 61]
  [org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint request GenericEndpoint.java 116]
  [org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleRequest RemoteEndpoint.java 257]
  [org.eclipse.lsp4j.jsonrpc.RemoteEndpoint consume RemoteEndpoint.java 186]
  [org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer handleMessage StreamMessageProducer.java 188]
  [org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer listen StreamMessageProducer.java 90]
  [org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor run ConcurrentMessageProcessor.java 95]
  [java.util.concurrent.Executors$RunnableAdapter call Executors.java 511]
  [java.util.concurrent.FutureTask run FutureTask.java 266]
  [java.util.concurrent.ThreadPoolExecutor runWorker ThreadPoolExecutor.java 1149]
  [java.util.concurrent.ThreadPoolExecutor$Worker run ThreadPoolExecutor.java 624]
  [java.lang.Thread run Thread.java 748]]}

```

**Comment by agzam** [comment]
Apparently I just needed to create a proper .lsp/config.edn. I did this:

```
{"dependency-scheme" "jar"
 "project-specs" [{:project-path "project.clj"
                   :classpath-cmd ["lein" "classpath"]}]}  
```

And it seems to be working with the latest version installed via brew --HEAD. 

Perhaps a better way of letting the user know about missing required config options need to be considered, minimal example configs for deps, lein and boot posted in readme and brew formula updated.
